src/style.css

```diff
@@ -0,0 +1,103 @@
+/* Fonts */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+/* Search bar */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments). Fix all cases